system/modules/core/templates/elements/ce_accordion.html5

```diff
@@ -25,7 +25,7 @@
         <?php endif; ?>
 
         <?php if ($this->caption): ?>
-          <figcaption class="caption"><?php echo $this->caption; ?></figcaption>
+          <figcaption class="caption" itemprop="caption description"><?php echo $this->caption; ?></figcaption>
```

Comment:
I think it is wrong to use both `caption` and `description`. Isn't the description of an image the content of the `alt` attribute?